app/views/banner/BannerView.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+import React from 'react'
+import {
+	View,
+	TouchableHighlight,
+  Image,
+} from 'react-native';
+
+var WebWrapper = require('../WebWrapper');
+var css = require('../../styles/css');
+
+// Card for displaying a large banner on the top of the screen
+export default class BannerView extends React.Component {
+  gotoWebView = () => {
+    // launch website
+    this.props.navigator.push({ id: 'WebWrapper', component: WebWrapper, title: this.props.site.title, webViewURL: this.props.site.url });
```

Comment:
@srkirkland  Given the complexity of the props, do you want to add PropTypes to the BannerView class?